test_requirements.txt

```diff
@@ -14,6 +14,8 @@ idna==3.4
 importlib-metadata==4.2.0
 iniconfig==1.1.1
 mccabe==0.7.0
+mypy==0.971
```

Comment:
This definitely works, but mypy 1.0.0 is out now if you wanted to give it a try.

Reply:
Ah I just "pip install mypy"'d in a docker container running Python 3.6 (https://github.com/JoshData/python-email-validator/blob/main/test_requirements.txt) and that's what I got. It's probably latest version supporting Python 3.6. I'll update to Python 3.7.

Reply:
Good call!